src/components/ToolMenu/FeatureInfo/FeatureInfoTabs/index.tsx

```diff
@@ -0,0 +1,156 @@
+import React, {
+  useEffect,
+  useState
+} from 'react';
+
+import {
+  Tabs,
+  TabsProps
+} from 'antd';
+
+import OlFeature from 'ol/Feature';
+import OlLayerVector from 'ol/layer/Vector';
+import OlSourceVector from 'ol/source/Vector';
+
+import {
+  Tab
+} from 'rc-tabs/lib/interface';
+
+import MapUtil from '@terrestris/ol-util/dist/MapUtil/MapUtil';
+
+import useMap from '@terrestris/react-geo/dist/Hook/useMap';
+
+import {
+  PropertyFormItemReadConfig,
+  PropertyFormTabConfig
+} from '@terrestris/shogun-util/dist/model/Layer';
+
+import useHighlightVectorLayer from '../../../../hooks/useHighlightVectorLayer';
+
+import FeatureInfoForm from '../FeatureInfoForm';
+import PaginationToolbar from '../PaginationToolbar';
+
+import './index.less';
+
+export type FeatureInfoTabsProps = TabsProps & {
+  features: OlFeature[];
+  layerName: string;
+  tabConfig?: PropertyFormTabConfig<PropertyFormItemReadConfig>[];
+};
+
+export const FeatureInfoTabs: React.FC<FeatureInfoTabsProps> = ({
+  features,
+  layerName,
+  tabConfig,
+  ...passThroughProps
+}) => {
+  const [currentPage, setCurrentPage] = useState<number>(1);
+  const [selectedFeature, setSelectedFeature] = useState<OlFeature>();
+
+  const map = useMap();
+
+  const vectorLayerName = `selection-layer-${layerName}`;
+
+  useHighlightVectorLayer({
+    layerName: vectorLayerName
+  });
+
+  useEffect(() => {
+    if (!map) {
+      return;
+    }
+
+    const vectorLayer = MapUtil.getLayerByName(map, vectorLayerName) as OlLayerVector<OlSourceVector>;
+
+    if (!vectorLayer) {
+      return;
+    }
+
+    vectorLayer.getSource()?.clear();
+    setCurrentPage(1);
+
+    if (features.length > 0) {
+      setSelectedFeature(features[0]);
+    }
+  }, [features, map, vectorLayerName]);
+
+  useEffect(() => {
+    if (!selectedFeature || !map) {
+      return;
+    }
+
+    const vectorLayer = MapUtil.getLayerByName(map, vectorLayerName) as OlLayerVector<OlSourceVector>;
+
+    if (!vectorLayer) {
+      return;
+    }
+
+    vectorLayer.getSource()?.clear();
+    vectorLayer.getSource()?.addFeature(selectedFeature);
+  }, [selectedFeature, map, vectorLayerName]);
+
+  const onChange = (page: number) => {
+    setCurrentPage(page);
+    setSelectedFeature(features[page - 1]);
+  };
+
+  if (!tabConfig || !selectedFeature) {
+    return <></>;
+  }
+
+  const exportFilter = (propertyName: string) => {
```

Comment:
just as a hint for the future, it should also be possible to find out if any `propertyName` in the array matches with `Array.some()`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

but this is also absolutely fine and should work just as well and doesn't have to be changed